servicetalk-http-api/src/main/java/io/servicetalk/http/api/HttpExecutionStrategy.java

```diff
@@ -60,4 +59,32 @@ default boolean hasOffloads() {
      * @return Merged {@link HttpExecutionStrategy}.
      */
     HttpExecutionStrategy merge(HttpExecutionStrategy other);
+
+    /**
+     * Return An execution strategy which contains only the additional offloads present in other that were not present
+     * in this execution strategy.
+     *
+     * @param other An execution strategy
+     * @return An execution strategy which contains only the additional offloads present in other that were not present
+     * in this execution strategy.
+     */
+    default HttpExecutionStrategy missing(HttpExecutionStrategy other) {
```

Comment:
This method isn't used, what is the intended use case for this method? does this need to be an instance method? If not can we move this into a static factory method to keep this interface focused on state?

Ideally we could do the same with `merge` after we remove special case combinations.

Reply:
why does it need to be an instance method? can we defer adding it until it is used to have a more clear picture on intended use and how it will replace `diff`?